### ISSUE TITLE
Preguntas Control

### ISSUE THREAD

**Issue by mpalaciosc** [RodrigoToroIcarte/IIC2113-2023-1#25]
Holaa, van a subir (o "liberar") las respuestas del control de clean code cap 1 y 2? Ya que en canvas me sale el puntaje pero las respuestas salen bloqueadas

**Comment by RodrigoToroIcarte** [comment]
Sí. Igual vamos a discutir las preguntas durante la clase de hoy.
